.github/workflows/Build.yml

```diff
@@ -717,3 +717,17 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./ci
+
+  #64
+  openlane-linux:
+    runs-on: "ubuntu-20.04"
```

Comment:
```suggestion
    needs: ["magic-linux", "netgen-linux", "openroad-linux", "yosys-linux-py37", "yosys-linux-py38"]
    runs-on: "ubuntu-20.04"
```

Reply:
Perhaps let's wait for these jobs to finish so that the packages built in the same CI can be used during testing.